test/library/packages/TOML/test/commentTest.compopts

```diff
@@ -0,0 +1 @@
+-M ../../../modules/packages/
```

Comment:
Oops. Missing a blank line here.

Reply:
Thanks @Rapiz1 - This shouldn't impact the compopts format, but would impact a `.good`.